contracts/ColonyNetworkAuction.sol

```diff
@@ -0,0 +1,203 @@
+/*
+  This file is part of The Colony Network.
+
+  The Colony Network is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  The Colony Network is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with The Colony Network. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+pragma solidity ^0.4.17;
+pragma experimental "v0.5.0";
+pragma experimental "ABIEncoderV2";
+
+import "../lib/dappsys/math.sol";
+import "./ColonyNetworkStorage.sol";
+import "./ERC20Extended.sol";
+import "./IColony.sol";
+
+
+contract ColonyNetworkAuction is ColonyNetworkStorage {
+  event AuctionCreated(address auction, address token, uint256 quantity, uint256 startPrice);
+
+  function startTokenAuction(
+    address _token,
+    uint256 _quantity,
+    uint256 _startPrice)
+    public 
+    {
+    ERC20Extended clny = ERC20Extended(IColony(_colonies["Common Colony"]).getToken());
```

Comment:
Is there a reason I'm not spotting to cast to `ERC20Extended` only to cast it to `address` below?

Reply:
No good reason. Missed that when cleaning up